Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -4022,13 +4033,24 @@ public async Task<bool> TryLogWarningAsync(SandboxedProcessOutput standardError,
                 return false;
             }
 
-            FormatOutputAndPaths(
+            bool errorWasTruncated = false;           

```

Comment:
>error [](start = 17, length = 5)

This function deals with warnings, not errors

Reply:
Changed to warningWasTruncated